app/model/recommender.py

```
@@ -26,4 +26,47 @@ def generate_recommendations(user_df, brand_df, model, dataset, user_features, t
                 "updated_at": datetime.now(timezone.utc)
             })
 
+    return pd.DataFrame(results)
+
+import numpy as np
```

Ruff (0.12.2)

Redefinition of unused `np` from line 1

Remove definition: `np`

(F811)